client/packages/requisitions/src/RnRForms/api/hooks/useRnRFormContext.ts

```diff
@@ -1,32 +1,41 @@
 import React from 'react';
 import { RnRFormLineFragment } from '../operations.generated';
-import { create, RecordWithId } from '@openmsupply-client/common';
+import {
```

Comment:
Needed to scroll in footer, so added ref and scrolling here.

Also wanted cached rnr draft to be very lean, so created foundbyids